google/cloud/pubsub_v1/subscriber/scheduler.py

```diff
@@ -162,7 +163,25 @@ def shutdown(
                 work_item = self._executor._work_queue.get(block=False)
                 if work_item is None:  # Exceutor in shutdown mode.
                     continue
-                dropped_messages.append(work_item.args[0])  # type: ignore[index]
+
+                dropped_message = None
+                if sys.version_info < (3, 14):
```

Comment:
Could you add a `TODO: Clean up once we only support Python 3.14+`? Just so we have an easier time finding this in 4 years so we can clean up the code.